### ISSUE TITLE
Debugger: map funktion keys to often used actions (step, and to on...)

### ISSUE THREAD

**Issue by MarcusDenker** [guillep/pharo-core#3729]
Due to problems with the key code handling this is not easy to do now, but when this is cleaned up, we should provide default mapping of Function keys (F1-Fn) for the debugger's most used
actions (step, step into, step over and so on)

**Comment by estebanlm** [comment]
Hi @MarcusDenker 
While I agree with this need, it is not an achievable goal in the short term, since the VM does not maps those keycodes. 
But the new headless VM will solve that problem, maybe there it can be reconsidered (and in that case, please use well common key associations and not "just someones" :P)

**Comment by MarcusDenker** [comment]
Yes, agreed... I wonder if we should tag these issue somehow "this is here but only so that we do not forget"

**Comment by dvtomas** [comment]
Hi,
I'm new to Pharo. Remapping debugger keys was one of the first things i've tried, current mapping seem just horrible to me (I'm used to Intellij Idea - F7/F8/F9 etc). Do I understand correctly that this is being blocked by the VM not supporting Fn keys at all? Sounds like a shame, the function keys are present on the keyboards for maybe 50 years now..

**Comment by pavel-krivanek** [comment]
[kbFix.zip](https://github.com/pharo-project/pharo/files/6686262/kbFix.zip)

we use this remapping

**Comment by guillep** [comment]
This should be easily doable now
